app.py

```
@@ -0,0 +1,300 @@
+        cmd = [
+            sys.executable,
+            'convert_directory.py',
+            source_path,
+            '--output', output_path,
+            '--to', output_format
+        ]
+
+        if verbose:
+            cmd.append('--verbose')
+
+        result = subprocess.run(
```

Ruff (0.13.0)

`subprocess` call: check for execution of untrusted input

(S603)